package.json

```diff
@@ -8,13 +8,13 @@
   },
   "bugs": "https://github.com/TryGhost/Ghost/issues",
   "private": true,
-  "version": "0.5.10",
+  "version": "0.6.0",
   "dependencies": {
-    "ghost": "0.5.10",
+    "ghost": "latest",
```

Comment:
I’d like to keep this explicitly set to the latest version number, so that people have control over when they upgrade their heroku apps to use any new release as they see fit. I don’t want people to ever be surprised to find a deploy upgraded something on them that the didn’t want.